src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -22,7 +21,7 @@ public class FindCommandParser implements Parser<FindCommand> {
      * @throws ParseException if the user input does not conform the expected format
      */
     public FindCommand parse(String args) throws ParseException {
-        FindCommand findCommand = null;
+        FindCommand findCommand;
```

Comment:
I'm abit lost about what the purpose of this statement is, wouldnt leaving it undeclared imply that findCommand is assigned to null too? 

Sorry :(

Reply:
They are the same. I just removed the initializer to make it less redundant :)